libut/uthash.pc.in

```diff
@@ -0,0 +1,9 @@
+prefix=/usr
+libdir=LIBDIR
+includedir=/usr/include
+
+Name: uthash
+Description: C macros for hash tables and more 
+Version: VERSION
+Libs: -L${libdir} -lut
+Cflags: -I${includedir}/uthash
```

Comment:
Can you explain a bit about what kind of package manager requires this file? and maybe reduce this pull request to *only* those changes, with a nice verbose commit message explaining all the stuff I currently don't understand? (Feel free to rebase and force-push changes into your branch; that will automatically update the pull request on GitHub.)